### ISSUE TITLE
Multiple Crosshairs

### ISSUE THREAD

**Issue by ZacRoden** [mpstark/SPT-SimpleCrosshair#4]
Seems to be an issue with mulitple crosshairs, first raid works, but ever raid after that adds an additional crosshair that doesnt seem to go away.
![Xk8l3wT](https://github.com/user-attachments/assets/941b00d9-9d76-4a9a-9480-c5cf733c31c8)

**Comment by AfxousaAjagara** [comment]
also experiencce this isue
![image](https://github.com/user-attachments/assets/60164139-4597-4549-a526-e2cb2ec3e9d5)

**Comment by stckytwl** [comment]
The DestroyCrosshair method in Plugin.cs appears to be destroying only the SimpleCrosshair component and not the gameObject along with it. As shown in the left image under the "components" sub-menu:

|![](https://github.com/user-attachments/assets/4c5dce0f-bf84-4ee3-a05d-eac3501ee813)<br>Inactive crosshair|![](https://github.com/user-attachments/assets/b63eb47f-9fd8-4688-8ee8-90de79ba2d7a)<br>Active crosshair|
|:-:|:-:|

**Comment by mpstark** [comment]
Yeah looks like a refactoring bug that wasn't caught because the EFTBattleUIScreen is now not destroyed like it had been. Sorry again, haven't been working with EFT stuff in a bit.

**Comment by mpstark** [comment]
Should be fixed in latest.
